### PR TITLE
Update wrong closing tag order

### PR DESCRIPTION
**Pull request by symera**
Closing tags for div, fieldset and form were mixed up. The two closing divs at the end had no purpose.

### CODE REVIEW

File: AP_Broadcast_Email.php [hunk 1]
```diff
@@ -200,13 +200,11 @@
 							<p class="submittop">
                 <input type="submit" name="confirm" value="Continue to Confirmation" tabindex="3" />
```
Comment (franzliedke) [inline AP_Broadcast_Email.php]: Can you also add the missing closing paragraph tag here?

**Comment by franzliedke** [comment]
Thanks for the PR! Please add a description next time so that it is easier to understand why these changes are necessary. ;)

**Comment by symera** [comment]
I added the missing paragraph tag as you suggested (thanks for pointing that out).
The rest is just some small changes with the tabs/indents.

**Comment by franzliedke** [comment]
Awesome, thank you!

**Pull request merged**